### PR TITLE
Software Development Kit subsection of the Explanations section

### PR DESCRIPTION
**Pull request by nibble0101**
# Improve the Introduction and Explanation sections project

| Section | Sub-section | Technical Writer |
| --- | --- | --- |
|Explanations | Software Development Kit  |  Chris Estepa |

This  is **Software Development Kit** sub-section of the **Explanations** section

image
image
image

### CODE REVIEW

**Review by proudofsimin — APPROVED** [review]
Thank you

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Perfect PR : clean commit history, properly based, relevant screenshots, PR good in scope.

**Comment by Rohitesh-Kumar-Jain** [comment]
@chris-4444 I would like you to once go through the PR, and approve when you think it's ready to get merged.

**Review by chris-4444 — APPROVED** [review]
LGTM.

File: docusaurus/docs/explainations/software-development-kit.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Software Development Kit
+---
+
+![Wechaty: Conversational RPA SDK for Chatbot Makers](/img/wechaty-logo.svg)
```
Comment (lijiarui) [inline docusaurus/docs/explainations/software-development-kit.md]: Maybe we change to SDK image better than wechaty logo here.

File: docusaurus/docs/explainations/software-development-kit.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Software Development Kit
```
Comment (lijiarui) [inline docusaurus/docs/explainations/software-development-kit.md]: The title is `Software Development Kit` but you talk about `Software Development Kit` as well as `conversational UI and AI`, `RPA`, etc, so I think you should either change the title or change the content here.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: docusaurus/docs/explainations/software-development-kit.md (hunk 1 above)
Comment (lijiarui) [reply docusaurus/docs/explainations/software-development-kit.md]: ping @nibble0101

**Review by lijiarui — CHANGES_REQUESTED** [review]
ping @nibble0101

File: docusaurus/docs/explainations/software-development-kit.md (hunk 1 above)
Comment (nibble0101) [reply docusaurus/docs/explainations/software-development-kit.md]: Removed the image

File: docusaurus/docs/explainations/software-development-kit.md (hunk 2 above)
Comment (nibble0101) [reply docusaurus/docs/explainations/software-development-kit.md]: Removed content not related to SDK

**Comment by nibble0101** [comment]
@chris-4444  I have removed all the other sub-sections except the one talking about SDK. Would you mind reading through again to see if it is still okay?

**Review by chris-4444 — APPROVED** [review]
@nibble0101 @lijiarui Updated the title to match the contents.

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Review by lijiarui — CHANGES_REQUESTED** [review]
image

It looks strange here since the title is tooooooo long.

And we already have an RPA explanation here: http://wechaty.js.org/docs/explanations/rpa So we needn't explain RPA at this part twice.

I suggest we merge them together or just delete this part.

**Review by chris-4444 — APPROVED** [review]
@lijiarui I have shortened the title. Let us retain this section as it was previously a section in the old Wechaty site and is also important. We just have it put in one section to make it look more organized. The sidebar would also be updated and would look somehow like this: 
image

**Review by chris-4444 — APPROVED** [review]
Removed RPA.

File: docusaurus/docs/explanations/software-development-kit.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: SDK and Conversational Interfaces
```
Comment (lijiarui) [inline docusaurus/docs/explanations/software-development-kit.md]: I suggest changing to `SDK & CUI`

File: docusaurus/sidebars.ts [hunk 4]
```diff
@@ -336,6 +336,7 @@ const explanations = {
     'explanations/lifecycle',
     'explanations/alternatives',
     'explanations/glossary',
+    'explanations/software-development-kit',
```
Comment (lijiarui) [inline docusaurus/sidebars.ts]: change `software-development-kit` to `sdk-cui`, making it more unified.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: docusaurus/docs/explanations/software-development-kit.md (hunk 3 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/software-development-kit.md]: Changed title to SDK & CUI

File: docusaurus/sidebars.ts (hunk 4 above)
Comment (nibble0101) [reply docusaurus/sidebars.ts]: Changed file name to sdk-cui,

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Requested changes have been implemented

**Review by lijiarui — APPROVED** [review]
LGTM

**Pull request merged**